### DISCUSSION TITLE
Add padding around PDF page?

### DISCUSSION THREAD

**Discussion by LonelessCodes** [Hopding/pdf-lib#1028] [Q&A]
I'm trying to make a simple web app to add additional whitespace to presentations from our professors, so I can take notes there.
I want to be able to add spacing at the top, right, left or bottom, but I only know to add space to the right and the bottom, via `page.setWidth()` and `page.setHeight()`.

Can someone help me figure out how I can add space to the left or top of a page or alternatively move the contents of a page to the right or bottom? Much appreciated.

**Comment by abdullahqureshi5050** [comment]
did to find any answer?

**Comment by akomm** [comment]
Existing PDF or new one you generate yourself from scratch
